### PR TITLE
Include table with links to other repositories

### PR DESCRIPTION
**Pull request by Ein-Tim**
This PR adds a table with the links to other repos to the README.md file.
This is already done i.e. in the iOS-Repo: 
https://github.com/corona-warn-app/cwa-app-ios#repositories

### CODE REVIEW

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@Ein-Tim Thanks!

File: README.md [hunk 1]
```diff
@@ -92,6 +92,33 @@ The German government has asked SAP and Deutsche Telekom AG to develop the Coron
...
+| ------------------- | --------------------------------------------------------------------- |
+| [cwa-documentation] | Project overview, general documentation, and white papers.            |
+| [cwa-app-ios]       | Native iOS app using the Apple/Google exposure notification API.      |
+| [cwa-app-android]   | Native Android app using the Apple/Google exposure notification API.  |
+| [cwa-wishlist]      | Community feature requests.                                           |
+| [cwa-website]       | The official website for the Corona-Warn-App                          |
+| [cwa-server]        | Backend implementation for the Apple/Google exposure notification API.|
+| [cwa-verification-server] | Backend implementation of the verification process.             |
+| [cwa-verification-portal] | The portal to interact with the verification server             |
+| [cwa-verification-iam]    | The identy and access management to interact with the verification server |
```
Comment (MikeMcC399) [inline README.md:107]: Typo
identy => identity

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Please correct typo.

Also I wonder about the two non-listed repositories:
https://github.com/corona-warn-app/cwa-backlog and
https://github.com/corona-warn-app/cwa-hotline

**Comment by Ein-Tim** [comment]
@MikeMcC399 
Thank you for your comment. 
I will correct the typo later 👍
I've left the backlog and the hotline repo out since they both seem rather inactive...
Should that change, we can add them later on

File: README.md (hunk 1 above)
Comment (Ein-Tim) [reply README.md]: Done

**Comment by dsarkar** [comment]
@Ein-Tim nice: 

image


@MikeMcC399 Thanks for review

**Pull request merged**